front/src/pages/settings/data-model/SettingsObjects.tsx

```diff
@@ -54,6 +37,19 @@ export const SettingsObjects = () => {
   const theme = useTheme();
   const navigate = useNavigate();
 
+  const activeMetadataObjects = useRecoilValue(activeMetadataObjectsSelector);
```

Comment:
Could we move this to a hook that would expose the metadata state without having the external users to know about the state names and internal implementation.

I would love to have something like: const { activeObjects, disableObjects ) = useObjectMetadata() that is internally doing these calls

Reply:
Done!